src/generator/AutoRest.Ruby/Templates/MethodTemplate.cshtml

```diff
@@ -120,7 +120,7 @@ def @(Model.Name)_async(@(Model.MethodParameterDeclaration))
   @EmptyLine
 }
 
-  path_template = '@Model.Url'
+  path_template = '@((Model.Url.StartsWith("/") ? Model.Url.TrimStart('/') : (string)Model.Url))'

```

Comment:
Does this change rely on information that `base_url` where the path is going to be appended must end with `/` ?